### DISCUSSION TITLE
help with removing EEPROM for custom Orin Nano carrier with Scarthgap

### DISCUSSION THREAD

**Discussion by arrow53** [OE4T/meta-tegra#1700] [Q&A] [answered]
I need to zero out the cvb entry `cvb_eeprom_read_size = <0x0>;` with file `tegra234-mb2-bct-misc-p3767-0000.dts`

there are some tips [here](https://github.com/OE4T/meta-tegra/discussions/1171) but I'm struggling to sort it out.

I made a custom layer with `meta-eeprom/recipes-bsp/tegra-binaries/tegra-bootfiles_%.bbappend` that has 
```
SRC_URI += "file://tegra234-mb2-bct-misc-p3767-0000.dts"

do_install:append() {
  install -d ${D}${datadir}/tegraflash
  install -m 0644 ${UNPACKDIR}/tegra234-mb2-bct-misc-p3767-0000.dts ${D}${datadir}/tegraflash/
}

FILES:${PN} = "${datadir}/tegraflash/tegra234-mb2-bct-misc-p3767-0000.dts"
```

But, I get this error even though my custom version of the dts file is in my `files` directory.

```
ERROR: /yocto/orin-nano-scarthgap-sd/layers/meta-tegra/recipes-bsp/tegra-binaries/tegra-bootfiles_36.3.0.bb: Unable to get checksum for tegra-bootfiles SRC_URI entry tegra234-mb2-bct-misc-p3767-0000.dts: file could not be found

```

**Comment by dwalkes** [comment, marked as answer]
> But, I get this error even though my custom version of the dts file is in my files directory.

See suggestions in https://docs.yoctoproject.org/dev-manual/layers.html#following-best-practices-when-creating-layers and in https://docs.yoctoproject.org/ref-manual/variables.html#term-FILESEXTRAPATHS

It seems you are missing `FILESEXTRAPATHS:prepend` as discussed there.

**Reply by arrow53** [reply]
thanks @dwalkes yeah, I had thought `SRC_URI` would look in `files` but guess not.

For anyone that comes across this. I ended up doing two recipes:

`tegra-binaries_%.bbappend` gets the file.

```
FILESEXTRAPATHS:prepend := "${THISDIR}/files:"

SRC_URI += "file://tegra234-mb2-bct-misc-p3767-0000.dts"

```

Then I move into place with `tegra-bootfiles_%.bbappend`
```
do_install:append() {
  install -m 0644 ${S}/../sources-unpack/tegra234-mb2-bct-misc-p3767-0000.dts ${D}${datadir}/tegraflash/
}
```

I think I could do a better job with the relative file locations / names, but this does work.
